projects/plugins/boost/app/modules/critical-css/class-critical-css.php

```diff
@@ -652,31 +654,76 @@ public function get_critical_css() {
 	 * @see style_loader_tag
 	 */
 	public function asynchronize_stylesheets( $html, $handle, $href, $media ) {
+		// If is AMP, do not alter the stylesheet loading.
 		if ( function_exists( 'is_amp_endpoint' ) && is_amp_endpoint() ) {
 			return $html;
 		}
 
+		// If there is no critical CSS, do not alter the stylesheet loading.
 		if ( false === $this->get_critical_css() ) {
 			return $html;
 		}
 
-		if ( ! apply_filters( 'jetpack_boost_async_style', true, $handle ) ) {
+		$available_methods = array(
+			'instant'  => 'media="' . $media . '"',
```

Comment:
I am very much inclined to change this to `default` since it is the default stylesheet loading behavior.